spec/factories/providers.rb

```diff
@@ -40,6 +40,7 @@
     opted_in { true }
 
     transient do
+      changed_at           { nil }
```

Comment:
should it be transient?

Reply:
I needs to be for what happens on line 67 in the `after(:create)` block below.

Reply:
brain thought

`create(:provider, changed_at: time.now.utc)`

Comments 63 to 66 must make some sense, once its part of a chain of thought...

Reply:
eh?